### PR TITLE
Palak_Assignment19_(40min+10+5+5+10)Min

### PR DESCRIPTION
**Pull request by PalakSJun20**
image
image
image

### CODE REVIEW

File: src/palakSJun20/DifferIndexFromArray.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	void findFirstDifferIndex(int[] array1 , int[] array2){
+		
+		if (array1.length == array2.length) {
+			for (int i = 0; i < array1.length; i++) {
+					if(array1[i] != array2[i]) {
+						System.out.println("Values are not matching at index " + i);
+						
+				}
+			}
+		}
```
Comment (KrishnaKTechnocredits) [inline src/palakSJun20/DifferIndexFromArray.java]: Where is the else block ???

File: src/palakSJun20/DifferIndexFromArray.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	void findMaxValueAtDifferIndex(int[] array5, int[] array6){
+		if (array5.length == array6.length) {
+			for (int i = 0; i < array5.length; i++) {
+					if(array5[i] != array6[i]) {
+						System.out.println("Values are not matching at index " + i);
+						if(array5[i] > array6[i])
+							System.out.println("From "+ "(" + array5[i] +","+ array6[i]  + ")" + " Max Value is" + array5[i]);
+						else 
+							System.out.println("From "+ "(" + array5[i] +","+ array6[i]  + ")" + " Max Value is " + array6[i]);
+				}
```
Comment (KrishnaKTechnocredits) [inline src/palakSJun20/DifferIndexFromArray.java:51]: Where is the else block ???

File: src/palakSJun20/DifferIndexFromArray.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+	
+	//Find max value at differed index.
+	void findMaxValueAtDifferIndex(int[] array5, int[] array6){
+		if (array5.length == array6.length) {
+			for (int i = 0; i < array5.length; i++) {
+					if(array5[i] != array6[i]) {
+						System.out.println("Values are not matching at index " + i);
+						if(array5[i] > array6[i])
+							System.out.println("From "+ "(" + array5[i] +","+ array6[i]  + ")" + " Max Value is" + array5[i]);
+						else
```
Comment (KrishnaKTechnocredits) [inline src/palakSJun20/DifferIndexFromArray.java]: The printing statement is not as expected.
System.out.println("Values are not matching at index -> " + index + ", From (" + arr1[index] + ","
							+ arr2[index] + ") max value is " + max);
You should write this line after if block.

if (arr1.length == arr2.length) {
			for (int index = 0; index < arr1.length; index++) {
				int max = arr2[index];
				if (arr1[index] != arr2[index]) {
					if (arr1[index] > arr2[index])
						max = arr1[index];
					System.out.println("Values are not matching at index -> " + index + ", From (" + arr1[index] + ","
							+ arr2[index] + ") max value is " + max);
				}
			}
		} else
			System.out.println("Both array's length is different.");

File: src/palakSJun20/PrimeNumberVariation.java [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+			}
+			if (flag == true) {
+				System.out.println("Number "+number+ " is not prime");
+			}
+			else 
+				System.out.println("Number "+ number+ " is Prime");
+		}
+		
+		//Print Total count of prime number b/w 2-100
+		int printTotalCountOfPrimeNumber(){
```
Comment (KrishnaKTechnocredits) [inline src/palakSJun20/PrimeNumberVariation.java]: Why this separate method is required ? You should directly write count and some logic in findGivenNumberPrimeOrNot() method.  Take two instance variable.

File: src/palakSJun20/PrimeNumberVariation.java [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+				}
+				if(flag == false){
+					count++;
+				}
+			}
+			return count;
+		}
+		
+		//Find sum of all prime numbers between 2->100
+		int findSumOfPrimeNumbers(){
```
Comment (KrishnaKTechnocredits) [inline src/palakSJun20/PrimeNumberVariation.java]: Not required.

**Review by KrishnaKTechnocredits — CHANGES_REQUESTED** [review]
..

File: src/palakSJun20/DifferIndexFromArray.java (hunk 1 above)
Comment (PalakSJun20) [reply src/palakSJun20/DifferIndexFromArray.java]: Updated with Else block

File: src/palakSJun20/DifferIndexFromArray.java (hunk 2 above)
Comment (PalakSJun20) [reply src/palakSJun20/DifferIndexFromArray.java:51]: updated with else block

File: src/palakSJun20/DifferIndexFromArray.java (hunk 3 above)
Comment (PalakSJun20) [reply src/palakSJun20/DifferIndexFromArray.java]: as per suggested changes has been updated.

File: src/palakSJun20/PrimeNumberVariation.java (hunk 4 above)
Comment (PalakSJun20) [reply src/palakSJun20/PrimeNumberVariation.java]: written sum and average logic in same method where we are counting prime number from 2-100

File: src/palakSJun20/PrimeNumberVariation.java (hunk 5 above)
Comment (PalakSJun20) [reply src/palakSJun20/PrimeNumberVariation.java]: Method Removed

**Pull request merged**